ci/do_ci.sh

```diff
@@ -32,7 +34,7 @@ fi
 export ENVOY_SRCDIR=$SOURCE_DIR
 
 # google cloud build times out when using full throttle.
-export NUM_CPUS=10
+export NUM_CPUS=5
```

Comment:
don't merge this before the discussion of private pool runners and maybe we can remove this restriction altogether